go.mod

```diff
@@ -3,10 +3,10 @@ module google.golang.org/grpc
 go 1.11
 
 require (
-	github.com/cncf/udpa/go v0.0.0-20191209042840-269d4d468f6f
-	github.com/envoyproxy/go-control-plane v0.9.4
+	github.com/cncf/udpa/go v0.0.0-20200629203442-efcf912fb354
+	github.com/envoyproxy/go-control-plane v0.9.7
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.4.2
```

Comment:
The go.mod change and vet.sh will conflict with #3932 
The other PR is mostly generated code, and has a cleaner change in vet.sh, maybe wait for that to merge first?

Reply:
Yes, will wait for it.

Reply:
And even after that, merging go.mod/go.sum won't be pleasant I guess. I would try first revert those changes, then merge and `go mod tidy`.

Reply:
I had to add a few more lines to `vet.sh` from the ones added in #3932 because my changes here do not have the corresponding pb.go changes. So, there were more complaints about usage of deprecated fields/types. We probably should do a scrub of the list once #3932 gets in.